software/edge/src/aeris_orchestrator/aeris_orchestrator/mavlink_adapter.py

```
@@ -151,6 +151,32 @@ def close(self) -> None:
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0,
+                )
+            self._logger(
+                "Sent MAV_CMD_NAV_RETURN_TO_LAUNCH to "
+                f"{self._host}:{self._port}"
+            )
+            return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)